src/App.js

```diff
@@ -241,80 +243,82 @@ class App extends Component {
     return (
       <ThemeProvider theme={themeChainsfr}>
         <Provider store={store}>
-          <SnackbarProvider
-            action={[
-              <IconButton key='close' aria-label='Close' color='inherit'>
-                <CloseIcon />
-              </IconButton>
-            ]}
-          >
-            <ConnectedRouter history={history}>
-              <Switch>
-                <LoginLayoutSwitch
-                  path={paths.login}
-                  isolate
-                  component={userIsNotAuthenticated(LoginContainer)}
-                />
-                <DefaultLayout
-                  exact
-                  path={paths.home}
-                  component={userIsAuthenticated(LandingPage)}
-                />
-                <DefaultLayout
-                  exact
-                  path={paths.wallet}
-                  component={userIsAuthenticated(WalletContainer)}
-                />
-                <DefaultLayout
-                  isolate
-                  path={`${paths.directTransfer}`}
-                  component={userIsAuthenticated(DirectTransferContainer)}
-                />
-                <DefaultLayout
-                  path={`${paths.transfer}`}
-                  isolate
-                  component={userIsAuthenticated(TransferContainer)}
-                />
-                <DefaultLayout
-                  path={`${paths.receive}`}
-                  isolate
-                  component={userIsAuthenticated(ReceiveContainer)}
-                />
-                <DefaultLayout
-                  path={`${paths.cancel}`}
-                  isolate
-                  component={userIsAuthenticated(CancelContainer)}
-                />
-                <DefaultLayout
-                  path={`${paths.contacts}`}
-                  component={userIsAuthenticated(RecipientsContainer)}
-                />
-                <DefaultLayout
-                  path={`${paths.accounts}`}
-                  component={userIsAuthenticated(AccountsManagementContainer)}
-                />
-                <DefaultLayout
-                  path={`${paths.receipt}`}
-                  isolate
-                  component={userIsAuthenticated(ReceiptContainer)}
-                />
-                <DefaultLayout
-                  path={`${paths.OAuthRedirect}`}
-                  component={userIsAuthenticated(OAuthRedirectComponent)}
-                />
-                <DefaultLayout
-                  path={`${paths.userSetting}`}
-                  component={userIsAuthenticated(UserSettingContainer)}
-                />
-                {process.env.REACT_APP_ENV === 'test' && (
+          <PersistGate loading={null} persistor={persistor}>
```

Comment:
This part of changes does not seem to belong to this issue.

Reply:
fixed